### ISSUE TITLE
My two cents on new naval units

### ISSUE THREAD

**Issue by Hanstheslayer** [Anuken/Mindustry-Suggestions#1675]
**Describe the content or mechanics you are proposing.**
I would like to start by admitting that I am not a Mindustry expert, so maybe my suggestion can end beign overpowered/useless. Also, english isn't my mother language, so I would like to say sorry if I made any mistake.

I have been searching if this idea was already posted, and it kind of was, however I would like to add my two cents on it. What I think could be good for the game would be to add a new "logistical" branch for the naval units, in a way similar to the "mono" branch of air troops. The idea behind it would be to transport units or blocks across the map without having to invest too much on aerial units, with the disadvantage that it would be limited to water zones. Also, it would add some strategical depth, since you could coordinate water to land assults. As I see it, it would be like this:

Tier 1 would be a basic transport unit, wich could transport 2 - 4 m^2 of cargo and maybe ~40 materials. It could have some basic weaponry, like the one on the alpha core mech.

Tier 2 would add a missile weapon, and increase the cargo capacity.

Tier 3 would keep getting more cargo capacity, and install a small shield projector like on the tier 5 "mono" unit (I don't remember the name).

Tier 4 would ditch the shield, but transform the ship into a submarine esque unit, which could remain hidden if moving through deep water.

Tier 5 would add a new weapon, a big but slow missile similar to an ICBM, minus the abusable range. It could resemble a nuclear submarine.

As a bonus, sea mines to protect the waters from the enemy. They could work like shock mines, and remain hidden from the enemy.

TLDR: Water transport units, from 1st. to 3rd. tier normal boats, and 4 - 5 submarines. Also seamines.

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.
This would allow:**

- A cheaper, but more situational, way to transport payloads across the map, be it units or buildings.

- More strategical depth, since attacks could come from the sea.

- More uses for land units on maps with too much water without having to use aerial transport.

- Maybe it could make water transport routes, which I think could be interesting to see in game, while having to protect said routes from an enemy wanting to disrupt the supply chain...

**Before making this issue, place an X in the boxes below to confirm that you have acknowledged them. Failure to do so may result in your request being closed automatically.**

  X - _I have done a quick search in the list of suggestions to make sure this has not been suggested yet._
 X - _I have checked the Trello to make sure my suggestion isn't planned or implemented in a development version._
  X - _I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet._
  X - _I have read README.md to make sure my idea is not listed under the "A few things you shouldn't suggest" category._

**Comment by Bohaska** [comment]
Anuke already expressed his interest in support naval units. Let him finish that before making this suggestion

**Comment by sample-text-here** [comment]
how would naval units pickup/place cargo

**Comment by Hanstheslayer** [comment]
My bad, I didn't know that anuke was already on it, since I don't have a discord. Thanks for the information.

**Comment by Hanstheslayer** [comment]
We could make a variant of the payload conveyor, calles a "dock", used to pickup/drop units from these units

**Comment by catatagun** [comment]
Support naval should be unit factory
